packages/utils/src/object.ts

```diff
@@ -294,6 +295,39 @@ function normalizeValue(value: any, key?: any): any {
   return value;
 }
 
+/**
+ * Decycles an object to make it safe for json serialization.
+ *
+ * @param obj Object to be decycled
+ * @param memo Optional Memo class handling decycling
+ */
+function decycle(obj: any, memo?: Memo): any {
+  if (memo === undefined) {
+    // tslint:disable-next-line:no-parameter-reassignment
+    memo = new Memo();
+  }
+  if (
```

Comment:
`if (isArray(obj) || isPlainObject(obj))` would work better here imo.

Reply:
Test fail with:
```
RangeError: Maximum call stack size exceeded
          at new WeakSet (<anonymous>)
          at new Memo (/Users/haza/Projects/sentry-javascript/packages/utils/src/memo.ts:12:92)
          at decycle (/Users/haza/Projects/sentry-javascript/packages/utils/src/object.ts:259:110)
          at Object.<anonymous> (/Users/haza/Projects/sentry-javascript/packages/utils/src/object.ts:283:228)
          at JSON.stringify (<anonymous>)
          at Object.safeNormalize (/Users/haza/Projects/sentry-javascript/packages/utils/src/object.ts:292:32)
          at Object.<anonymous> (/Users/haza/Projects/sentry-javascript/packages/utils/test/object.test.ts:400:31)
          at Object.asyncFn (/Users/haza/Projects/sentry-javascript/node_modules/jest-jasmine2/build/jasmine_async.js:82:37)
          at resolve (/Users/haza/Projects/sentry-javascript/node_modules/jest-jasmine2/build/queue_runner.js:52:12)
          at new Promise (<anonymous>)
```
if I do this